### ISSUE TITLE
Conduct Inventory of Furniture and Equipment at London Office

### ISSUE THREAD

**Issue by AmyCYF** [CodeYourFuture/Ops-Programme#192]
**Description:**
We need a comprehensive inventory of all furniture and equipment at the London office, including items like chairs, desks, tables, and other essentials. This inventory will help with space planning, resource management, and future budgeting needs. Each item should be listed with details such as quantity, condition, and any immediate repair or replacement needs.

**Acceptance Criteria:**
-
- [ ] Itemized Inventory: Document each item type (e.g., chairs, desks, tables) along with a count of items in usable condition.
- [ ] Inventory Document: Compile all findings into a single, accessible document or spreadsheet for easy reference by team members.

**Comment by cynthimicallef101** [comment]
@AmyCYF is this still required since we are moving to the new offices on Sunday?

**Comment by AmyCYF** [comment]
Yes, it would be good to know what we have in the office as it helps us plan how many people we can faciliate. At the moment we do not have a consolidated list of the tables, chairs, white boards, tvs etc.

**Comment by cynthimicallef101** [comment]
Ok thank you
